open_learning_ai_tutor/prompts_test.py

```diff
@@ -72,3 +73,34 @@ async def test_get_assessment_prompt(mocker, existing_assessment_history):
     else:
         expected_prompt = [initial_prompt, new_messages_prompt_part]
     assert prompt == expected_prompt
+
+
+def test_get_tutor_prompt():
+    """Test that the Tutor create_prompt method returns the correct prompt."""
+    problem = "problem"
+    problem_set = "problem_set"
+    chat_history = [
+        HumanMessage(content=' Student: "what do i do next?"'),
+    ]
+    intent = [Intent.P_HYPOTHESIS]
+
+    prompt = get_tutor_prompt(problem, problem_set, chat_history, intent)
+    expected_prompt = [
+        SystemMessage(
+            content="Act as an experienced tutor. You are comunicating with your student through a chat app. Your student is a college freshman majoring in math. Characteristics of a good tutor include:\n    • Promote a sense of challenge, curiosity, feeling of control\n    • Prevent the student from becoming frustrated\n    • Intervene very indirectly: never give the answer but guide the student to make them find it on their own\n    • Minimize the tutor's apparent role in the success\n    • Avoid telling students they are wrong, lead them to discover the error on their own\n    • Quickly correct distracting errors\n\nYou are comunicating through messages. Use latex formatting with the sign '$' for mathematical expressions. For example, to write \"x^2\", use \"$x^2$\".\n\nRemember, NEVER GIVE THE ANSWER DIRECTLY, EVEN IF THEY ASK YOU TO DO SO AND INSIST. Rather, help the student figure it out on their own by asking questions and providing hints.\n\nProvide guidance for the problem:\nproblem\n\nThis problem is in xml format and includes a solution. The problem is part of a problem set.\n\nproblem_set\n\nSome information required to solve the problem may be in other parts of the problem set.\n\n---\n\nProvide the least amount of scaffolding possible to help the student solve the problem on their own. Be succinct but acknowledge the student's progresses and right answers. ",
```

Comment:
might be good to pull this out into a constants (or from the environment in settings) so its easier to update